bsmetadata/metadata_processors.py

```diff
@@ -40,9 +40,21 @@ class MetadataConfig:
         default=" |||",
         metadata={"help": "The character sequence that is used to separate all global metadata from the actual text."},
     )
+    entity_setting: str = field(
+        default="normal",
+        metadata={"help": "The settings in which you want to use entites. Valid choices: (beg, end, normal)"},
+    )
     max_seq_len: int = field(
         default=512, metadata={"help": "The maximum number of tokens to use for each training chunk."}
     )
+    local_metadata_special_token_start: Optional[Dict[str, str]] = field(
+        default=None,
+        metadata={"help": "A dictionary whose keys correspond to a local metadata type and values ...."},
```

Comment:
It seems to me that this description (that I had proposed to you, sorry :see_no_evil: ) is not finished. Would you mind finishing it?